herc/config.py

```diff
@@ -0,0 +1,53 @@
+import pyhocon
+import threading
+import traceback
+import sys
+import importlib
+
+_config = None
```

Comment:
Had another thought about this - putting config loading into `__init__.py` means that the config file will be loaded (and thus required) even during testing. Does that strike you as icky?